tests/rustdoc-json/reexport/pub_use_doc_hidden.rs

```diff
@@ -6,8 +6,9 @@ mod repeat_n {
     pub struct RepeatN {}
 }
 
-/// not here
+/// is here
```

Comment:
I don't think this is the right behavior. If the underlying item isn't part of the public API than I don't think the import of it should be either.

Reply:
The import is publicly accessible so I don't see why not?

Reply:
1. Ideologically: `RepeatN` has asked to not be part of the public API. We don't show it in the HTML docs.
2. Practicly: Theirs very little use to consumers telling them "You import something here, but you don't get to know what it it". If it's a part of the API, we should tell them the full story. If it isn't then we shouldn't acknowledge the import at all. What purpose does this weird in-between serve? (If theirs a compelling use case for just knowing this, that'd easily change my mind)